tests/drivers/gpio/gpio_more_loops/Kconfig

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+config NUMBER_OF_LOOPBACKS
+	int "Number of GPIO loopbacks"
+	default 3
+	help
+		This test must be run on setups that have 'gpio_loopback' fixture.
```

Comment:
help text indent is 1x tab followed by 2x spaces

Reply:
Fixed.